docs/network_firewall.rst

```diff
@@ -1,29 +1,26 @@
 Set Up the Network Firewall
 ===========================
 
-Now that you've set up your password manager, you can move on to setting
-up the Network Firewall. You should stay logged in to the *Admin Workstation*
-to access the Network Firewall's web interface for configuration.
+Now that you've set up your password manager, you can move on to setting up
+the Network Firewall. You should stay logged in to the *Admin Workstation* to
+access the Network Firewall's web interface for configuration.
 
 Unfortunately, due to the wide variety of firewalls that may be used, we
 do not provide specific instructions to cover every type or variation in
-software or hardware.
-However, if you have the necessary
-expertise, we provide `abstract firewall rules`_ that can be
-implemented with iptables, Cisco IOS etc.
-This guide is based on pfSense, and assumes your
-firewall hardware has at least three interfaces: WAN, LAN, and OPT1. For
-hardware, you can build your own network firewall (not covered in this
-guide) and `install
-pfSense <https://doc.pfsense.org/index.php/Installing_pfSense>`__ on it.
-For most installations, we recommend buying a dedicated firewall
-appliance with pfSense pre-installed, such as the one recommended in the
-Hardware Guide.
-
-We currently recommend the `pfSense SG-4860
-<https://store.netgate.com/SG-4860.aspx>`__, which has 6 interfaces: WAN,
-LAN, OPT1, OPT2, OPT3 and OPT4. If your firewall only has 3 NICs (WAN,
-LAN, and OPT1), you will need to use a switch on the OPT1 interface
+software or hardware. However, if you have the necessary expertise, we
+provide `abstract firewall rules`_ that can be implemented with iptables, Cisco
+IOS etc. This guide is based on pfSense, and assumes your firewall hardware has
+at least three interfaces: WAN, LAN, and OPT1. For hardware, you can build
+your own network firewall (not covered in this guide) and `install pfSense 
+<https://doc.pfsense.org/index.php/Installing_pfSense>`__ on it. For most
+installations, we recommend buying a dedicated firewall appliance with
+pfSense pre-installed, such as the one recommended in the
+:ref:`Hardware Guide <hardware_guide>`.
+
+We currently recommend the `pfSense SG-3100
+<https://store.netgate.com/SG-3100.aspx>`__, which has 6 interfaces: WAN,
```

Comment:
nit: It actually has 3 interfaces/NICs (WAN, OPT, and LAN) but it does indeed have 6 ports (WAN, OPT and LAN1 through LAN4)